SpatialGDK/Documentation/content/cross-server-rpcs.md

```diff
@@ -65,10 +68,26 @@ The tables below show where cross-server RPCs are executed based on where they w
 
 #### Invoking a cross-server RPC from a client-worker
 
-The call is not processed because this type of RPC is only for a server-worker instance to call.
+The SpatialOS Runtime does not process the call because this type of RPC is only available to a server-worker, not a client-worker.
 
+### Unresolved parameters in an RPC
+
+In some situations, a server-worker or client-worker might receive an RPC with a parameter that it cannot resolve: for example, an actor passed as a parameter may not be checked out by the receiver. When this happens, the GDK outputs a warning to the logs. However, there may be circumstances in which you don't want the GDK to output a warning because the behavior is acceptable; when this is the case, you can disable the warnings on a per-RPC basis, either using C++ or Blueprints.
+
+#### How to disable warnings for unresolved parameters in an RPC (using C++)
+
+To disable these warnings on an RPC in C++, add an `AllowUnresolvedParameters` tag to the `UFUNCTION` macro of the RPC function.
+
+```
+UFUNCTION(Server, AllowUnresolvedParameters, Reliable)
+void MyServerRPC();
+```
+
+#### How to disable warnings for unresolved parameters in an RPC (using Blueprints)
+
+To disable these warnings on an RPC in a Blueprint, in the **Details** panel of the event, click the eye in the top-right, and make sure **Show All Advanced Details** is ticked. The **Allow Unresolved Parameters** field in the panel will appear.
```

Comment:
Quick format and made it use standard Microsoft Manual of Style terms - using US English, y'all!.
I assume it defaults to unchecked.
```suggestion
To disable these warnings on an RPC in a Blueprint:
1.  In the **Details** panel of the event, select the eye in the top-right.
2. Check the **Show All Advanced Details** option. 
The panel now displays the **Allow Unresolved Parameters** field.
```